### PR TITLE
i18n(es): Update `index.mdx` & `frontmatter.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

- What does this PR change? Give us a brief description.
Updated `index.mdx & `frontmatter.mdx` based on this commit 7bf0f20cdc8403e79220c4b8c5d3e42fb0d1672b

### CODE REVIEW

File: docs/src/content/docs/es/reference/frontmatter.md [hunk 1]
```diff
@@ -134,6 +134,26 @@ interface HeroConfig {
 }
 ```
 
+### `banner`
+
+**type:** `{ content: string }`
```
Comment (Waxer59) [inline docs/src/content/docs/es/reference/frontmatter.md:139]:
```suggestion
**tipo:** `{ content: string }`
```

File: docs/src/content/docs/es/reference/frontmatter.md [hunk 2]
```diff
@@ -134,6 +134,26 @@ interface HeroConfig {
...
+
+El valor `content` puede incluir HTML para enlaces u otro contenido.
+Por ejemplo, esta página muestra un banner que incluye un enlace a `example.com`.
+
+```md
+---
+title: Página con un banner
+banner:
+  content: |
+    We just launched something cool!
```
Comment (Waxer59) [inline docs/src/content/docs/es/reference/frontmatter.md:151]: tienes que borrar esta linea

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo unas sugerencias!

File: docs/src/content/docs/es/reference/frontmatter.md (hunk 2 above)
Comment (delucis) [reply docs/src/content/docs/es/reference/frontmatter.md:151]:
```suggestion
```

**Review by delucis — APPROVED** [review]
Thank you @dreyfus92 and for the speedy review @Waxer59! 🚀

**Pull request merged**